packages/resolvers/zod/index.ts

```diff
@@ -0,0 +1,29 @@
+import { ZodFirstPartyTypeKind, ZodObject } from "zod";
+import type { ResolvedField, ResolvedValidator } from "@astro-reactive/common";
+
+export function zodResolver(schema: ZodObject<any>) {
```

Comment:
Add a return type here, `ResolvedField` I think. Also, some JSDoc for the function would be nice!

Then I will ask that you start a new page in our docs site as well for resolvers. I don't mind if it is very initial. Just copy-paste the JSDoc 😄

Reply:
Alright, will do it! 👍